src/search-scenario/azext_search_scenario/requests.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import json
+
+from ._clierror import ScenarioSearchError
+from .constants import MatchRule, SearchScope, SEARCH_SERVICE_URL
+
+
+def get_search_result_from_api(search_keyword, scope=SearchScope.All, match_rule=MatchRule.All, top_num=5):
+    '''Search related e2e scenarios'''
+    import requests
+
+    payload = {
+        "keyword": search_keyword,
+        "scope": scope,
+        "match_rule": match_rule,
+        "top_num": top_num,
+    }
+
+    response = requests.post(SEARCH_SERVICE_URL, json.dumps(payload))
+    if response.status_code != 200:
+        raise ScenarioSearchError(
+            f"Failed to connect to '{SEARCH_SERVICE_URL}' with status code "
```

Comment:
The error message is not so accurate because it may successfully connect with the service, but there is server side error.

Reply:
Done